pkg/kubelet/rkt/rkt.go

```diff
@@ -935,6 +942,12 @@ func (r *Runtime) preparePod(pod *api.Pod, pullSecrets []api.Secret) (string, *k
 		newUnitOption("Service", "KillMode", "mixed"),
 	}
 
+	if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.SELinuxOptions != nil {
+		opt := pod.Spec.SecurityContext.SELinuxOptions
+		selinuxContext := fmt.Sprintf("%s:%s:%s:%s", opt.User, opt.Role, opt.Type, opt.Level)
```

Comment:
I checked on coreos distro, the `/var/lib/rkt/`, `/var/lib/rkt/pods`, and `/var/lib/rkt/pods/$uuid` all have the same context (`system_u:object_r:var_lib_t:s0`). So I am going just to take the context of the `/var/lib/rkt`